src/main/resources/templates/smallfull/review.html

```diff
@@ -16,6 +16,157 @@ <h1 id="page-title-heading" class="govuk-heading-l">Check your answers</h1>
+                            currentAmount = *{profitAndLoss?.profitOrLossBeforeTax?.interestReceivableAndSimilarIncome?.currentAmount},
+                            previousAmount = *{profitAndLoss?.profitOrLossBeforeTax?.interestReceivableAndSimilarIncome?.previousAmount}
+                        )">
+          </div>
+
+          <div th:replace="fragments/review/balanceSheetValueRow :: balanceSheetValueRow (
+                            description = 'Interest payable and similar charges',
+                            identifier = 'interest-payable-and-similar-charges',
+                            currentAmount = *{profitAndLoss?.profitOrLossBeforeTax?.interestPayableAndSimilarCharges?.currentAmount},
+                            previousAmount = *{profitAndLoss?.profitOrLossBeforeTax?.interestPayableAndSimilarCharges?.previousAmount}
+                        )">
+          </div>
+
+          <div th:replace="fragments/review/balanceSheetValueRow :: balanceSheetValueRow (
+                            description = 'Profit (or loss) before tax',
+                            identifier = 'profit-or-loss-before-tax',
+                            currentAmount = *{profitAndLoss?.profitOrLossBeforeTax?.totalProfitOrLossBeforeTax?.currentAmount},
+                            previousAmount = *{profitAndLoss?.profitOrLossBeforeTax?.totalProfitOrLossBeforeTax?.previousAmount},
+                            isTotalRow = true
+                        )">
+          </div>
+
+        </th:block>
+
+        <th:block
+            th:if="*{profitAndLoss?.profitOrLossForFinancialYear?.totalProfitOrLossForFinancialYear?.currentAmount != null or profitAndLoss?.profitOrLossForFinancialYear?.totalProfitOrLossForFinancialYear?.previousAmount != null}">
+
+          <div th:replace="fragments/review/balanceSheetValueRow :: balanceSheetValueRow (
+                            description = 'Tax',
+                            identifier = 'tax',
+                            currentAmount = *{profitAndLoss?.profitOrLossForFinancialYear?.tax?.currentAmount},
+                            previousAmount = *{profitAndLoss?.profitOrLossForFinancialYear?.tax?.previousAmount}
+                        )">
+          </div>
+
+          <div th:replace="fragments/review/balanceSheetValueRow :: balanceSheetValueRow (
+                            description = 'Profit (or loss) for the financial year',
+                            identifier = 'profit-or-loss-before-tax',
+                            currentAmount = *{profitAndLoss?.profitOrLossForFinancialYear?.totalProfitOrLossForFinancialYear?.currentAmount},
+                            previousAmount = *{profitAndLoss?.profitOrLossForFinancialYear?.totalProfitOrLossForFinancialYear?.previousAmount},
+                            isTotalRow = true
+                        )">
+          </div>
+
+        </th:block>
+
+      </dl>
+
+    </div>
+
+    <!-- End Profit and Loss
```

Comment:
Have you closed off the comment properly here?

Reply:
Good catch, addressed